scripts/common/preflights.sh

```diff
@@ -233,3 +234,15 @@ function kotsadm_prerelease() {
         fi
     fi
 }
+
+function host_nameservers_reachable() {
+    if [ "$AIRGAP" = 1 ]; then
+        return 0
+    fi
+    if ! discover_non_loopback_nameservers; then
+        printf "\n${YELLOW}No nameservers configured on the host are accessible by pods. All DNS queries outside the cluster will fail. Continue?${NC} "
```

Comment:
@areed can you help me understand how i'd want to answer this?  if i'm installing an application (the primary kURL use case today), in an airgapped environment, what does this mean?  If I see this, should I continue or abort?

Reply:
In an airgapped environment this preflight is skipped. In an online environment you should abort and fix your nameserver.